components/sysutils/stress/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Adam Stevko. All rights reserved.
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=	 	stress
+COMPONENT_VERSION=	1.0.4
+COMPONENT_REVISION=	1
+COMPONENT_FMRI=		utilities/stress
```

Comment:
We haven't shipped anything in this top-level classification, please, choose something more common